src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewModel.swift

```diff
@@ -101,6 +107,15 @@ final class FooterViewModel {
 			self.isPrimaryButtonHidden = true
 			self.isSecondaryButtonHidden = true
 		}
+		updateInsets()
```

Comment:
Perhaps you could investigate when `update(to state: VisibleButtons)` is called. When it is called at initialization, perhaps we could spare also the call you created in the initializer. Just an idea, not sure about it.